manual/xpdf-utils/xpdf-utils.nuspec

```diff
@@ -3,21 +3,22 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>xpdf-utils</id>
-    <version>3.4.0.20170430</version>
-    <packageSourceUrl>https://github.com/Chocolatey/chocolatey-coreteampackages/tree/master/automatic/xpdf-utils</packageSourceUrl>
+    <version>4.04</version>
```

Comment:
As we have started using Chocolatey CLI v2.0.0, please change this to be the normalized version number (the same as Chocolatey CLI will be using).

That means, no leading zeros, and a minimum of 3 part version numbers.
In this case that will be `4.4.0`.

Reply:
Addressed in 762fcd31673d90e0f42df9cf3f827c0e4ae5c4a3.